server/monitor/src/main/java/org/apache/accumulo/monitor/next/Endpoints.java

```diff
@@ -52,11 +53,17 @@
 import org.apache.accumulo.monitor.next.SystemInformation.ProcessSummary;
 import org.apache.accumulo.monitor.next.SystemInformation.TableSummary;
 
-import io.micrometer.core.instrument.Meter.Id;
-import io.micrometer.core.instrument.cumulative.CumulativeDistributionSummary;
-
 @Path("/")
 public class Endpoints {
+  /**
+   * A {@code String} constant representing supplied resource group
+   */
+  private static final String GROUP = "group";
+
+  /**
+   * A {@code String} constant representing supplied table name.
```

Comment:
```suggestion
   * A {@code String} constant representing supplied tableId.
```

Reply:
Actually scratch that. These constants represent the parameter keys not an actual tableId or resource group. The javadoc should be updated to reflect that.

Reply:
Yes, I took it into account